lib/mix_test_watch/config.ex

```diff
@@ -21,6 +23,7 @@ defmodule MixTestWatch.Config do
     %__MODULE__{
       tasks:    get_tasks(),
       clear:    get_clear(),
+      include:  get_included(),
```

Comment:
I went ahead and modified this to ```include_extensions``` in the latest commit. Let me know your thoughts on this and if there's anything that you think would be more fitting. Thanks! :)